src/helpers/prefetch-wallet-card-images.ts

```diff
@@ -0,0 +1,16 @@
+import {Image} from 'react-native';
+
+import {Wallet} from '@app/models/wallet';
+import {getPatternName} from '@app/utils';
+
+export async function prefetchWalletCardImages() {
+  try {
+    await Promise.all(
```

Comment:
Do we have `Promise.allSettled` support?

Reply:
yes, it's better. Already replace to allSettled